plugins/modules/sefcontext.py

```diff
@@ -39,21 +41,28 @@
     default: a
   setype:
     description:
-    - SELinux type for the specified target.
+    - SELinux type for the specified I(target).
+    type: str
+  equal:
+    description:
+    - Path to use for substituting the I(target). The context labeling for the I(target) subtree is made equivalent to this path.
+    version_added: 5.9.0
```

Comment:
```suggestion
    version_added: 6.3.0
```

Reply:
Oops, was looking at changelog of wrong branch.